src/App.js

```diff
@@ -0,0 +1,37 @@
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import Navbar from "./components/navbar/navbar";
+import Home from './pages/Home/home';
+import SignOut from "./pages/SignOut/signout";
+import AddHouse from "./pages/House/addHouse";
+import DeleteHouse from "./pages/House/deleteHouse";
+import HouseDetails from "./pages/House/houseDetails";
+import ReserveForm from "./pages/Reservation/reserveForm";
+import SignUp from "./pages/Signup/signup";
+import Login from "./pages/login/login";
+import ProtectedRoute from "./components/route/protectedroute";
+import HomeReserve from "./pages/House/homeReserved";
+import MyReservation from "./pages/Reservation/myReservation";
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://user-images.githubusercontent.com/57421823/233378075-1e898f1f-ff58-464d-a1b9-7162a7ac8c3d.png)

That way, your code will comply with latest standards.  💪

Reply:
Changes have been made